website/docs/namespaced-mode.md

```diff
@@ -0,0 +1,116 @@
+---
+id: namespaced-mode
+title: Namespaced Mode
+sidebar_label: Namespaced Mode
+original_id: namespaced-mode
+---
+
+---
+
+### What is Namespaced Mode?
+
+Namespaced mode is the other end of the orchestration spectrum & is an antithesis of the admin-mode of operation. Here, the ChaosOperator as well as the chaos resources (chaosexperiment, chaosengine, chaosresult CRs and the experiment pods) are run in the same namespace where the application under test (AUT) resides. This mode serves the use-cases where the developers/users don't typically have much autonomy over cluster usage and operate strictly within a specific allotted namespace. However, in this case, too, the cluster-admin or an equivalent user with the right permissions are required to install them CRDs upfront.
+
+Note that this mode of operation restricts the scope of chaos experiments to those at the pod-level, as the node/infra level experiments need cluster-wide access.
+
+### How to use Namespaced Mode?
+
+In order to use Namespaced Mode, you just have to install the namespace-scoped litmus operator manifest in the desired (app) namespace. This installs the ChaosOperator, configured to watch for ChaosEngine resources in the same namespace while also setting up the RBAC that can be used for the execution of the standard (generic suite) of chaos experiments.
+
+#### Setup the Namespace-Scoped Chaos Operator
+
+- Install the chaos CRDs separately via an admin account
+
+  ```
+  kubectl apply -f https://raw.githubusercontent.com/litmuschaos/litmus/master/docs/litmus-namespaced-scope/litmus-namespaced-crds.yaml
+  ```
+
+- Install the namespace-scoped operator (& continue to operator subsequent steps) via the dev/user account
+
+  - Pull the manifest from [here](https://raw.githubusercontent.com/litmuschaos/litmus/master/docs/litmus-namespaced-scope/litmus-namespaced-operator.yaml)
+
+  - Replace the 'default' string with that of your desired namespace across the manifest
+
+  - Apply the modified operator manifest
+
+  ```
+  kubectl apply -f <namespace-scoped-operator.yaml>
+  ```
+
+#### Install the ChaosExperiment CR
+
+- The supported experiments in the namespaced-mode, today, are:
+
+  - pod-delete
+  - container-kill
+  - pod-cpu-hog
+  - pod-memory-hog
+  - pod-network-latency
+  - pod-network-loss
+  - pod-network-duplication
+  - pod-network-corruption
+
+- Pull the respective chaosexperiment custom resources from the [ChaosHub](https://hub.litmuschaos.io/generic)
+
+#### Prepare the ChaosEngine CR
+
+- Create a chaosengine, similar to the one shown below, by providing the appropriate application & experiment information.
+
+```yaml
```

Comment:
If this sample ChaosEngine CR is present at a GitHub repository it'd be great if you could add te embedmd link to it as well.

Reply:
The exact sample isn't present. The [closest](https://github.com/litmuschaos/chaos-charts/blob/master/charts/generic/pod-delete/engine.yaml) one I could find has different namespace/chaosServiceAccount.